libc/include/llvm-libc-macros/linux/sys-stat-macros.h

```diff
@@ -10,7 +10,7 @@
 #define __LLVM_LIBC_MACROS_LINUX_SYS_STAT_MACROS_H
 
 // Definitions from linux/stat.h
```

Comment:
kind of curious why we don't just simply

#include <linux/stat.h>

for this whole file?

Reply:
I think originally we defined all of these constants inside some spec file, so the original generated header files didn't have `#include` statements.  Then when they were refactored into the current .h file structure, they were just ported 1-to-1.  With the current include structure, we definitelt can do more than before.

Reply:
On linux, I guess it is possible for user to install `libc-dev` without `linux-headers`. However, I think it is fine as the kernel header package is also common enough.